docs/modules/ROOT/pages/developer.adoc

```diff
@@ -1,27 +1,29 @@
-= Developer
+= Managing API Keys
 :url-github-cloud-cli: https://github.com/hazelcast/hazelcast-cloud-cli/
 :url-github-go-sdk: https://github.com/hazelcast/hazelcast-cloud-sdk-go
 :url-cloud-developers: https://cloud.hazelcast.com/settings/developer
 :url-cloud-api: https://cloud.hazelcast.com/v1/api/explorer
 
-The Hazelcast Cloud allows you to manage your resources and clusters via link:{url-cloud-api}[API], link:{url-github-go-sdk}[SDK], and link:{url-github-cloud-cli}[CLI]. You can get further details about them from xref:api-reference.adoc[API Reference].
+Hazelcast Cloud allows you to manage your resources and clusters via an link:{url-cloud-api}[API], an link:{url-github-go-sdk}[SDK], and a link:{url-github-cloud-cli}[CLI]. To use these applications, you need an API key and an API secret, which can be created from link:{url-cloud-developers}[Developers] tab in your account or team settings. 
 
-In order to use these applications, you need to provide API Key and API Secret. This API Key and API Secret can be created from link:{url-cloud-developers}[Developers] tab in Settings as shown below. 
+== Prerequisites
+
+To manage API keys for a team's cluster, you must have either an admin or developer role. See xref:teams-and-users.adoc[].
```

Comment:
adoc extension in link text
![Screen Shot 2021-09-02 at 15 26 21](https://user-images.githubusercontent.com/1237982/131843328-a1786c6d-6904-4702-921c-8d19fcd48389.png)